src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,40 +86,60 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={query}
+              onChange={(event) => {
+                setQuery(event.target.value);
+              }}
             />
           </div>
-
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {movieNotFound && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              disabled={!buttonDisabled}
+              className={cn(
+                'button is-light disabled',
+                {
+                  'is-loading': isLoad,
+                },
+              )}
             >
               Find a movie
             </button>
           </div>
 
-          <div className="control">
-            <button
-              data-cy="addButton"
-              type="button"
-              className="button is-primary"
-            >
-              Add to the list
-            </button>
-          </div>
+          {foundMovie
+          && (
+            <div className="control">
+              <button
+                data-cy="addButton"
+                type="button"
+                className="button is-primary"
+                onClick={addMovieToList}
+              >
+                Add to the list
+              </button>
+            </div>
+          )}
         </div>
       </form>
-
-      <div className="container" data-cy="previewContainer">
+      <div
+        className={cn('container',
+          {
+            'is-hidden': !foundMovie,
+          })}
+        data-cy="previewContainer"
+      >
         <h2 className="title">Preview</h2>
-        {/* <MovieCard movie={movie} /> */}
+        <MovieCard movie={foundMovie} />
```

Comment:
do not render this component if you don't have a movie

Reply:
kk